radio/src/gui/common/stdlcd/radio_hardware.cpp

```diff
@@ -631,6 +634,15 @@ void menuRadioHardware(event_t event)
         else
           lcdDrawText(lcdNextPos, y, "---");
         break;
+#if defined(PCBX7) || defined(PCBXLITE) || defined(PCBX9LITE) || defined(PCBXLITES)
+       case ITEM_RADIO_HARDWARE_SMARTPORT_POWER:
+        g_eeGeneral.smartportUpdatePower = 1 - editCheckBox(1 - g_eeGeneral.smartportUpdatePower, HW_SETTINGS_COLUMN2, y, "SP power", attr, event);
```

Comment:
There is an issue when scrolling over the check box as it needs two clicks on the rotary instead of one